vipps-recurring-api.md

```diff
@@ -1139,11 +1160,15 @@ Example:
 
 Once a day, same as without variable amount.
 
-**Note:** In the examples above the `intervalCount` is 1. This can be changed as described in the [Intervals](#intervals) section.
+**Please note:** In the examples above the `intervalCount` is 1.
+This can be changed as described in the [Intervals](#intervals) section.
 
 #### Charge amount higher than the users max amount
 
-If the created charge is above the users `max amount`, the charge will be set to `DUE` with a `failureReason` as shown below. If the user does not update their maxAmount to the same or a higher amount than the charge, it will fail when `dueDate` + `retryDays` is reached.
+If the created charge is above the users `maxAmount`, the charge will be set
+to `PROCESSING`. If the user does not update their `maxAmount` to the same or a higher
+amount than the charge, it will fail when `due` + `retryDays` is reached, and
+the status will be `FAILED`.
```

Comment:
This is one of the cases where we will still be using failureReason, so I don't think this needs to be changed.

Reply:
I think we will use an event log.